charts/rds-auth-proxy/values.yaml

```diff
@@ -58,6 +58,8 @@ serviceAccount:
   # -- Annotations for the service-account - this can be used for IRSA
```

Comment:
Camelcase instead of kebab-case

Reply:
just removing the hyphen, this is for docs